docs/assert/false.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: false
+description: A boolean check. Passes if the first argument is false.
+categories:
+  - assert
+---
+
+## `false( state [, message ] )`
+
+A boolean check, inverse of `true()` equivalent to Chais's `assert.isFalse()`, and JUnit's `assertFalse()`. Passes if the first argument is false.
+
+| name               | description                          |
+|--------------------|--------------------------------------|
+| state              | Expression being tested              |
+| message (string)   | A short description of the assertion |
+
+### Description
+
+`false()` requires just one argument. If the argument evaluates to false, the assertion passes; otherwise, it fails. If a second message argument is provided, it will be displayed in place of the result.
```

Comment:
Would be good I think to have an extra line here that links to `true()`, see <https://api.qunitjs.com/assert/equal> for example which links to notEqual and strictEqual,

Reply:
Added a reference from `false.md` to `true.md` and one from `true.md` to `false.md`.